network_driver/Dockerfile.tendermint

```diff
@@ -0,0 +1,7 @@
+FROM tendermint/tendermint:develop
+WORKDIR /
+COPY ./tnode /tendermint
+COPY tendermint_entrypoint.sh tendermint_entrypoint.sh
+VOLUME /tmp/bigchain/tendermint /tendermint_node_data
```

Comment:
@muawiakh @shahbazn The volume mount is same for all nodes. This will be problematic right?

Reply:
these are the mount points for the container, you can attach it to different persistent volumes/storage classes. Shouldn't be a problem.